packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -126,6 +370,36 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
     CancellableStatusBar.show(execution, cancellationTokenSource);
     taskViewService.addCommandExecution(execution, cancellationTokenSource);
   }
+
+  public mkdirSyncRecursive(dirPath: string) {
```

Comment:
Why can't we use `mkdir -p` from shelljs?

Reply:
I introduced this code before introducing `shelljs` in the tests. This would make `shelljs` a runtime dependency. Right now it's only a dev dependency and used in tests only.

You ok with adding `shelljs` as runtime dep?

Reply:
`shelljs` is a runtime dep in sobjects-faux-generator package. @Sweetman is also adding it as a runtime dep in https://github.com/forcedotcom/salesforcedx-vscode/pull/310. There's a common need for it so I think it should be okay.

Reply:
Yeah. We already use it in https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-sobjects-faux-generator/package.json#L16 as well for its `mkdir -p` functionality. I would rather rely on it then write our own function which requires testing on all OS.

Reply:
I'll update the code then to use `shelljs`. Thanks!